### DISCUSSION TITLE
Possibility to search the Murmurations index by node name

### DISCUSSION THREAD

**Discussion by AltruistEnterprises** [MurmurationsNetwork/MurmurationsProtocol#49] [Ideas] [answered]
As the title suggests, I'm wondering if there is — or eventually could be — a way to search the index API for information about a particular node based on its name? 

This would be very useful for querying information about a particular node without always having the (correct) primary_url.

**Comment by geoffturk** [comment]
Currently there is no name search in the index but it is a possibility for a future parameter for indexing.

**Comment by jdaviescoates** [comment]
I'm pretty sure this is now implemented, no?

At least, it's certainly now possible to search by name on the Index Explorer, see https://test-tools.murmurations.network/index-explorer?schema=all&name=Sustainable&tags=food&tags_filter=or&tags_exact=false

**Comment by geoffturk** [comment, marked as answer]
Indeed, it is fully implemented now.
